lib/ansible/modules/network/netvisor/pn_fabric_local.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/python
+# Copyright: (c) 2018, Pluribus Networks
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = """
+---
+module: pn_fabric_local
+author: "Pluribus Networks (@rajaspachipulusu17)"
+version_added: "2.9"
+short_description: CLI command to modify fabric-local
+description:
+  - This module can be used to modify fabric local information.
+options:
+  pn_cliswitch:
+    description:
+      - Target switch to run the CLI on.
+    required: False
+    type: str
+  state:
+    description:
+      - State the action to perform. Use C(update) to modify the fabric-local.
+    required: True
```

Comment:
@rajaspachipulusu17 Thanks for raising the PR, but as mentioned in last PR's comment can u change the `state` param not to be the  mandatory param for the module, rather make any other param as mandatory and `required: True` which is actually required by the respective module to function.